frontend/src/features/contacts/contact/detail/Person.tsx

```diff
@@ -0,0 +1,197 @@
+              <Styled.RowAligned>
+                <Col md="4">{field.info}</Col>
+                <Col md="auto">
+                  <em>{field.label}</em>
+                </Col>
+              </Styled.RowAligned>
+            ))}
+          </Col>
+        </Styled.RowAligned>
+        <Styled.RowAligned>
+          <Col md={2}>
+            <strong>Phone:</strong>
+          </Col>
+          <Col>
+            {personPhoneNumbers.length === 0 && <span>N.A</span>}
+            {personPhoneNumbers.map((field: ContactInfoField, index: number) => (
+              <Styled.RowAligned>
+                <Col md="4">{phoneFormatter(field.info)}</Col>
+                <Col md="auto">
+                  <em>{field.label}</em>
+                </Col>
+              </Styled.RowAligned>
+            ))}
+          </Col>
+        </Styled.RowAligned>
+      </FormSection>
+      <FormSection key={person.id + '-Address'} className="mb-4">
+        <Styled.H2Primary>Address</Styled.H2Primary>
+        {personAddresses.map((field: AddressField, index: number) => (
+          <Styled.RowAligned className="pb-3">
+            <Col>
+              <div className="pb-2">
+                <strong>{field.label}</strong>
+                {field.address.addressType.id === AddressTypes.Mailing && (
+                  <em className="ml-4">from organization</em>
+                )}
+              </div>
+
+              <div>{field.address.streetAddress1}</div>
+              <div>{field.address.streetAddress2}</div>
+              <div>{field.address.streetAddress3}</div>
+              <div>
+                {field.address.municipality} {field.address.provinceCode}
+              </div>
+              <div>{field.address.postal}</div>
+              <div>{field.address.country}</div>
+              {index + 1 !== personAddresses.length && <hr></hr>}
+            </Col>
+          </Styled.RowAligned>
+        ))}
+      </FormSection>
+      <FormSection key={person.id + '-Comments'}>
+        <Styled.RowAligned>
+          <Col>
+            <div>
+              <strong>Comments:</strong>
+            </div>
+            {person.comment}
+          </Col>
+        </Styled.RowAligned>
+      </FormSection>
+    </>
+  );
+};
+
+function getContactInfo(
+  person: IContactPerson,
+  validTypes: Dictionary<string>,
+): ContactInfoField[] {
+  // Get only the valid types
+  let filteredFields = person.contactMethods.reduce(
+    (accumulator: ContactInfoField[], method: IContactMethod) => {
+      if (Object.keys(validTypes).includes(method.contactMethodType.id)) {
+        accumulator.push({
+          info: method.value,
+          label: validTypes[method.contactMethodType.id],
+        });
+      }
+      return accumulator;
+    },
+    [],
+  );
+
+  // Sort according to the dictionary order
+  return filteredFields.sort((a, b) => {
+    return Object.values(validTypes).indexOf(a.label) - Object.values(validTypes).indexOf(b.label);
```

Comment:
**nit**: given that `Object.values(validTypes)` is not going to change between invocations - would it be possible to pre-compute it so we don't have to generate it every time?

e.g.

```tsx
  const validValues = Object.values(validTypes);

  return filteredFields.sort((a, b) => {
    return validValues.indexOf(a.label) - validValues.indexOf(b.label)
  });


```